### ISSUE TITLE
Issue when opening a cluster containing other clusters

### ISSUE THREAD

**Issue by Pratik--Patel** [almende/vis#2561]
First of all, thanks for the great library!

I am using visjs network module.
When I try to programmaticaly open up a cluster containing other clusters, I get an exception from vis.js saying 'Cannot set property 'vx' of undefined'. The same issue is described here in this [SO question](http://stackoverflow.com/questions/36895691/nested-clusters-in-vis-js-networks-causing-problems)

I modified the potentially problematic source code to avoid such error by adding a check around them and the problem was fixed but I am not sure that this change will not introduce other problems in visjs. I updated the part of openClusters() method from this

```
            // old code
            // inherit speed
            _containedNode3.vx = clusterNode.vx;
            _containedNode3.vy = clusterNode.vy;

            // we use these methods to avoid re-instantiating the shape, which happens with setOptions.
            _containedNode3.setOptions({ hidden: false, physics: true });
```

to this 

```
                     // new code
                     // inherit speed
			if( clusterNode.vx != null && clusterNode.vy != null ){
				_containedNode3.vx = clusterNode.vx;
				_containedNode3.vy = clusterNode.vy;
			}

			if( _containedNode3 != null )
				// we use these methods to avoid re-instantiating the shape, which happens with  setOptions.
				_containedNode3.setOptions({ hidden: false, physics: true });
```

Is it a safe and clean way to fix this problem? Please let me know if there can be any better approach to this. Thanks!

**Comment by Tooa** [comment]
I get the following error when trying the provided jsbin:

> Uncaught ReferenceError: openClusters is not defined

Can you provide me a working example that shows the error? Then, I will look into the issue.

**Comment by Pratik--Patel** [comment]
I have modified and fixed the jsFiddle so that the error is demonstrated fine.

You can find it [here](http://jsfiddle.net/n3tqtzgo/10/)

**Comment by wimrijnders** [comment]
Fixed with #3330, closing for that reason.
